suricata/update/parsers.py

```diff
@@ -0,0 +1,164 @@
+import argparse
+from suricata.update import commands
+from suricata.update import config
```

Comment:
Please follow the PEP8 standards for writing code. These imports can be on a single line.

Reply:
2 new lines after imports.

Reply:
Sorry for the late reply, I was in a no network region for past few days.
Thanks for reviewing, will take care and address these changes in the next PR.